### PR TITLE
Support compressed requests always

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: misk/src/main/kotlin/misk/web/jetty/JettyService.kt [hunk 1]
```diff
@@ -167,12 +166,16 @@ class JettyService @Inject internal constructor(
...
     if (webConfig.gzip) {
-      gzipHandler.server = server
       gzipHandler.minGzipSize = webConfig.minGzipSize
       gzipHandler.addIncludedMethods("POST")
-      servletContextHandler.gzipHandler = gzipHandler
+    } else {
+      // GET is enabled by default for gzipHandler.
+      gzipHandler.addExcludedMethods("GET")
     }
+    gzipHandler.inflateBufferSize = 8192
```
Comment (tso) [inline misk/src/main/kotlin/misk/web/jetty/JettyService.kt]: I guess this was the missing piece to allow requests?

File: misk/src/test/kotlin/misk/web/GzipTest.kt [hunk 2]
```diff
@@ -9,82 +9,152 @@ import misk.web.jetty.JettyService
...
+    sink.use {
+      sink.writeUtf8("[\"")
+      sink.writeUtf8(miskHype(count).joinToString(separator = "\", \""))
+      sink.writeUtf8("\"]")
+    }
+
+    return buffer.readByteString().toRequestBody(MediaTypes.APPLICATION_JSON_MEDIA_TYPE)
+  }
+
+  class TestModule(val gzip: Boolean) : KAbstractModule() {
```
Comment (tso) [inline misk/src/test/kotlin/misk/web/GzipTest.kt]: cool

File: misk/src/main/kotlin/misk/web/jetty/JettyService.kt (hunk 1 above)
Comment (swankjesse) [reply misk/src/main/kotlin/misk/web/jetty/JettyService.kt]: yep.

Comment (swankjesse) [reply misk/src/main/kotlin/misk/web/jetty/JettyService.kt]: yep.

**Pull request merged**